lib/ernest.rb

```diff
@@ -62,7 +62,13 @@ def valid_key?
   get '/addresses' do
     content_type :json
 
-    page = Address.page(params[:page].to_i)
+    if params[:updated_since]
+      address = Address.where('updated_at >= ?', params[:updated_since])
```

Comment:
also slightly concerned here that there is no date parsing of the parameter to check validity. I could do `?updated_since=edballs` and cause application errors, presumably.